backend/composer/models.py

```diff
@@ -752,11 +752,14 @@ def validate_population_change(self):
                     "Cannot change population set after the statement has been exported."
                 )
 
+    def clean(self):
+        return self.validate_population_change()
+
     def save(self, *args, **kwargs):
         if not self.pk and self.sentence and not self.owner:
             self.owner = self.sentence.owner
 
-        self.validate_population_change()
+        self.clean()
```

Comment:
My preference would be to use a pre-save signal, but we can keep it like this as well